src/fprime_gds/executables/plotData.py

```diff
@@ -0,0 +1,146 @@
+#!/bin/env python3
+
+# ------------------------------------------------------------------------------------------
+# Program: Plot Data
```

Comment:
Does plotting data fall within the scope of what we want the fprime data product tools to do? There's lots of tools that exist for plotting time series data... I wonder if it might make more sense for the fprime data product tools to generate CSV files, allowing folks to plot their data in their favorite plotting tool.

Reply:
This code also looks specific to the `Ref/` app - maybe it would be a good example/guide to reference in the documentation instead?